canton/community/participant/src/main/scala/com/digitalasset/canton/participant/admin/PackageService.scala

```diff
@@ -417,6 +421,14 @@ class PackageService(
                 LoggingContextWithTrace(loggerFactory)
               )
             )
+      _ <- if (dryRun)
+              EitherT
+                .leftT[Future, Unit](
+                  PackageServiceErrors.Validation.DryRun.Error(mainPackage._1)
+                )
+                .leftWiden[DamlError]
+           else
+             EitherT.rightT[Future, DamlError](Future { () })
```

Comment:
Same loose note about the other service, I wouldn't initially expect this to be part of validation

Reply:
fixed